src/sentry/replays/query.py

```diff
@@ -590,6 +593,32 @@ class ReplayQueryConfig(QueryConfig):
     project = String(query_alias="project_id", is_filterable=False)
 
 
+class ReplaySubqueryConfig(QueryConfig):
```

Comment:
ah a separate query config is a great way to do this. nice.

Reply:
🎉